upf/upf.c

```diff
@@ -359,6 +359,29 @@ upf_init (vlib_main_t * vm)
 					FIB_SOURCE_PRIORITY_HI,
 					FIB_SOURCE_BH_SIMPLE);
 
+  clib_spinlock_init (&sm->upf_counter_lock);
```

Comment:
the look is not really needed.
Simple counters are per CPU anyway and if you don't use the index as per CPU id, then no locking is required.

Reply:
sure. will be fixed